### DISCUSSION TITLE
连接数异常

### DISCUSSION THREAD

**Discussion by Karmylr** [daeuniverse/dae#306] [Q&A]
使用`ls /proc/dae的pid/fd -l | grep socket: | wc -l`查看连接数，经常到几千，造成内存占用增大。但是openwrt的实时信息里却只有几百。怎么排查呢
![image](https://github.com/daeuniverse/dae/assets/69903376/124ebac5-5cc3-4653-97e3-9dc557b71dbe)
![image](https://github.com/daeuniverse/dae/assets/69903376/19f82923-375c-45b6-a991-c100a2b59af0)
![image](https://github.com/daeuniverse/dae/assets/69903376/dd56275c-04ed-4527-a733-d48d81cb220e)

**Comment by mzz2017** [comment]
用的什么版本？

**Reply by Karmylr** [reply]
daed 0.3.3-p1
